src/server.test.js

```diff
@@ -2,19 +2,39 @@ jest.mock('express')
 jest.mock('../sequelize/db')
 const express = require('express')
 const {getApp, startServer, stopServer} = require('./server')
+const app ={
+	use:()=>{},
+	patch:jest.fn(),
+	listen:()=>{}
+}
+express.mockReturnValue(app)
 const dbModule = require('../sequelize/db')
 
 dbModule.start = jest.fn()
-
+const userRoutes = require('./database_router')
+userRoutes.patch = jest.fn()
+describe('test users api for patch', () => {
+     beforeEach(() => {
+         jest.clearAllMocks()
+     })
+     it('should call patch function', () => {
+         startServer().then(() => {
+             app.patch.mock.calls[0][1]()
+             expect(userRoutes.patch).toHaveBeenCalled()
+         })
+     })
+ }) 
 describe('Testing the server', () => {
   test('getApp should return null when startServer has not been called', () => {
     const result = getApp() 
 
-    expect(result).toBe(null)
+    expect(result).toBe(app)
```

Comment:
`getApp` should return null if `startServer` has not been called(just like what the description says).